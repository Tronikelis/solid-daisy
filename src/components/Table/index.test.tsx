import { render } from "@solidjs/testing-library";
import { describe, expect, it } from "vitest";

import { Table } from ".";

describe("<Table />", () => {
    it("renders correctly", () => {
        const { container } = render(() => (
            <Table>
                <div>foo</div>
            </Table>
        ));

        expect(container).toMatchSnapshot();
    });
});
