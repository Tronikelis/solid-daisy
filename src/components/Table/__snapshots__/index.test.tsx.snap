// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<Table /> > renders correctly 1`] = `
<div>
  <div
    class="rounded-2xl bg-base-100 border border-gray-200 p-0 overflow-hidden"
  >
    <div
      class="overflow-x-auto"
    >
      <table
        class="table"
      >
        <div>
          foo
        </div>
      </table>
    </div>
  </div>
</div>
`;
